modules/webadmin.cpp

```diff
@@ -299,6 +299,8 @@ class CWebAdminMod : public CModule {
             WebSock.GetParam("appendtimestamp").ToBool());
         pNewUser->SetTimestampPrepend(
             WebSock.GetParam("prependtimestamp").ToBool());
+        pNewUser->SetBuiltinAuthDisabled(
+            WebSock.GetParam("builtinauthdisabled").ToBool());
```

Comment:
Same as above: Should users be able to edit this for themselves or should this be treated similar to `IsAdmin`?